--- tmva/src/RuleFitParams.cxx.orig	2025-02-14 14:47:05.706328347 +0000
+++ tmva/src/RuleFitParams.cxx	2025-02-14 14:48:05.158148747 +0000
@@ -24,6 +24,7 @@
  * (http://tmva.sourceforge.net/LICENSE)                                          *
  **********************************************************************************/
 
+#include <cmath>
 #include <iostream>
 #include <iomanip>
 #include <numeric>
@@ -880,7 +881,7 @@
       fstarVal = fRuleEnsemble->FStar(e);
       fFstar.push_back(fstarVal);
       fstarSorted.push_back(fstarVal);
-      if (isnan(fstarVal)) Log() << kFATAL << "F* is NAN!" << Endl;
+      if (std::isnan(fstarVal)) Log() << kFATAL << "F* is NAN!" << Endl;
    }
    // sort F* and find median
    std::sort( fstarSorted.begin(), fstarSorted.end() );
